File: c++/main3.cpp
class Base {
public:
    virtual void do_nothing() const = 0;
    virtual ~Base() {}
};

class Derived : public Base {
    void do_nothing() const {}
    ~Derived() {}
};

int main() {
    Base* obj = new Derived();
    for (int i = 0; i < 100000000; ++i) {
        obj->do_nothing();
    }
    delete obj;
    return 0;
}

File: swift/main3.swift
import Foundation

class Base {
    func doNothing() {}
}

class Derived : Base {
    override func doNothing() {}
}

let obj: Base = Derived()

let start = Date()

for _ in 0...100000000 {
    obj.doNothing()
}

let finish = Date()
let elapsed = finish.timeIntervalSince(start) * 1000

let result = "\(elapsed)\n"

print(result)
